content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+We talk about two distinct components in this doc - testgrids and release-informing/blocking/broken jobs. 
+All the release-informing/blocking/broken jobs jobs end up in the testgrid but we can also have jobs which are not release-informing/blocking/broken jobs ending up in the testgrid if they are added to the `/release/core-services/testgrid-config-generator/_allow-list.yaml`.
```

Comment:
Especially in the first paragraph of a doc it's important to keep jargon down and be clear. I'd suggest starting this like:

```
This document lays out the process of getting a CI job recorded as blocking or informing a release as well as the process for having job outputs exposed in TestGrid page. While all jobs that block or inform a release are necessarily exposed in TestGrid pages, it is not required that a job have influence on an OpenShift release for the job's output to be present in a TestGrid overview.
```